code/obj/machinery/door/airlock.dm

```diff
@@ -203,19 +210,30 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	operation_time = 6
 	brainloss_stumble = TRUE
 
-	get_desc()
+	get_desc(null, mob/user)
 		var/healthpercent = src.health/src.health_max * 100
 		switch(healthpercent)
 			if(90 to 99) //dont want to clog up the description unless it's actually damaged
-				. += "It seems to be in mostly good condition"
+				. += " It seems to be in mostly good condition"
 			if(75 to 89)
-				. += "It seems slightly [pick("dinged up", "dented", "damaged", "scratched")]"
+				. += " It seems slightly [pick("dinged up", "dented", "damaged", "scratched")]"
 			if(50 to 74)
-				. += "It looks [pick("busted", "damaged", "messed up", "dented")]."
+				. += " It looks [pick("busted", "damaged", "messed up", "dented")]."
 			if(25 to 49)
-				. += "It looks [pick("quite", "pretty", "rather", "notably")] [pick("mangled", "busted", "messed up", "wrecked", "destroyed", "haggard")]."
+				. += " It looks [pick("quite", "pretty", "rather", "notably")] [pick("mangled", "busted", "messed up", "wrecked", "destroyed", "haggard")]."
 			if(0 to 24)
-				. += "It is barely intact!"
+				. += " It is barely intact!"
+
+		if(src.operating == -1 && user.traitHolder.hasTrait("training_engineer")) // If the door is emagged and whoever is examining it has Engineer Training...
+			switch(demag_status)
+				if(UNSCREW_STAGE)
+					. += " <span class='notice'>The airlock wont respond to anything. A good way to start diagnosing this problem is by <b>unscrewing</b> the maintenance panel cover.</span>"
+				if(CUT_WIRE_STAGE)
+					. += " <span class='notice'>The wiring inside the airlock is completely fried! You'll have to <b>cut</b> all of it away before replacing it, though.</span>"
+				if(INSERT_WIRE_STAGE)
+					. += " <span class='notice'>The insides of the airlock is almost completely empty. You could place some new wiring with just about <b>5 lengths of cable coil</b>.</span>"
+				if(PULSE_STAGE)
+					. += " <span class='notice'>The repairs are almost done, you just have to <b>pulse</b> the mainboard to restart the airlock's circuitry. It's fallback system should take care of the rest.</span>"
```

Comment:
its vs it's

"its" is possessive, "it's" is a contraction of "it is". should be its